package.json

```diff
@@ -38,7 +38,7 @@
   "gitHead": "84e03109d9a590f9c8ef687c03d751f666080c6f",
   "readmeFilename": "README.md",
   "dependencies": {
-    "d3": "~3.5.0"
+    "d3": "^4.11.0"
```

Comment:
this should be `~3.5.0 || ^4.11`. Otherwise, there is no actual need to keep the code supporting d3 v3